Initial project governance (#20)
getsops/community: GOVERNANCE.md

  * ability to write quality code and/or documentation,
  * ability to collaborate with the team,
  * understanding of how the team works (policies, processes for testing and code review, etc),
  * understanding of the project's code base and coding and documentation style.

A new Maintainer must be proposed by submitting an [Issue to the community repository](https://github.com/getsops/community/issues). A simple majority vote of existing Maintainers
approves the application.  Maintainers nominations will be evaluated without prejudice to employer or demographics.

Maintainers who are selected will be granted the necessary GitHub rights,
and invited to the private maintainer mailing list.

### Removing a Maintainer

Maintainers may resign at any time if they feel that they will not be able to
continue fulfilling their project duties.

Maintainers may also be removed after being inactive, failure to fulfill their 
Maintainer responsibilities, violating the Code of Conduct, or other reasons.
Inactivity is defined as a period of very low or no activity in the project 
for a year or more, with no definite schedule to return to full Maintainer 
activity.

> A Maintainer may be removed at any time by a 2/3 vote of the remaining maintainers.

Reviewer: ... and here.

Depending on the reason for removal, a Maintainer may be converted to Emeritus
status.  Emeritus Maintainers will still be consulted on some project matters,
and can be rapidly returned to Maintainer status if their availability changes.

## CNCF Resources

> Any Maintainer may suggest a request for CNCF resources A simple majority of Maintainers approves the request.  The Maintainers

Reviewer: ```suggestion
Any Maintainer may suggest a request for CNCF resources.  A simple majority of Maintainers approves the request.  The Maintainers
```

may also choose to delegate working with the CNCF to non-Maintainer community
members, who will then be added to the [CNCF's Maintainer List](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
for that purpose.

## Code of Conduct

[Code of Conduct](./CODE_OF_CONDUCT.md)
violations by community members will be discussed and resolved
amongst maintainers. If a Maintainer is directly involved
in the report, the Maintainers will instead designate two Maintainers to work with the CNCF Code of Conduct Committee in resolving it.

## Voting

While most business in SOPS project is conducted by "[lazy consensus](https://community.apache.org/committers/lazyConsensus.html)", 
periodically the Maintainers may need to vote on specific actions or changes.
A vote can be taken for security or conduct matters. Any Maintainer may
demand a vote be taken.

Most votes require a simple majority of all Maintainers to succeed, except where
otherwise noted.  Two-thirds majority votes mean at least two-thirds of all 

> existing maintainers.

Reviewer: Does this mean that 2/3 of the maintainers have to participate in the vote, but the vote only needs a > 50% majority? Or does it mean that both 2/3 of the maintainers have to participate, and 2/3 of the votes need to be "accept"/"yes"?

I'd think the former, but I find the formulation not 100% clear (maybe also because I'm not a native English speaker), so we should probably be a little more precise here :)

Author: This section describes the different voting options that are used (either a simple majority [50.1%] or a two-thirds [2/3]). The default is majority unless noted otherwise. However, we are choosing a 2/3 supermajority as the number of votes that are needed as described in the various sections

Does that help provide some clarity?

> otherwise noted.  Two-thirds majority votes mean at least two-thirds of all 
> existing maintainers.

Reviewer: If I understand you correctly, this would be:
```suggestion
otherwise noted.  Two-thirds majority votes mean at least two-thirds of all 
existing maintainers have to participate in the vote;
of the cast votes, more than 50% have to be in favor.
```

> Most votes require a simple majority of all Maintainers to succeed, except where
> otherwise noted.  Two-thirds majority votes mean at least two-thirds of all 
> existing maintainers.

Reviewer: How about changing this to something like:
```suggestion
All votes require that the number of votes in favor of the motion exceeds a threshold.
In case of most votes, this is a simple majority of all persons that are entitled to
vote, which usually is the group of all Maintainers:
in these votes, the number of votes in favor of the motion must exceed half of the
total number of Maintainers.

In cases explicitly noted, the required number of votes in favor can be higher,
and/or only a subset of all Maintainers are entitled to vote.

Two-thirds majority votes (usually called "2/3 vote") require that the number of
votes in favor of the motion is at least 2/3 of the total number of persons entitled
to vote.
```

## Modifying this Charter

Changes to this Governance and its supporting documents may be approved by 

> a 2/3 vote of the Maintainers.

Reviewer: This applies here (it doesn't say "2/3 majority vote" here, only "2/3 vote")...

Author: The two are fundamentally the same. They both require more than 66.67% of the vote
